README.md

```diff
@@ -79,7 +79,7 @@ For more information please see `./CONTRIBUTING.md`.
 This library should always compile with any combination of features (minus
 `no-std`) on **Rust 1.41.1** or **Rust 1.47** with `no-std`.
 
-To build with the MSRV you will need to pin some dependencies:
+To build with the MSRV you will need to pin some dependencies (also for `no-std`):
```

Comment:
Sounds like we don't support `no_std` with those higher versions. Should say "also for 1.47"

Reply:
I agree this sentence is awkwardly worded. We can fix it later. But we do say in the above line that the MSRV for `no_std` is 1.47, so I think this does say the correct thing.